hw_30_plan/js/script.js

```diff
@@ -0,0 +1,132 @@
+const listItems = [
+    'Work out',
+    'Get some sleep',
+    'Call the boyfriend',
+    'Cook the dinner',
+    'Watch Netflix',
+    'Take a shower',
+    'Walk with the dog',
+]
+
+let ul = document.querySelector('#ul');
+
+let menu = document.querySelector('#menu');
+
+class Menu {
+    constructor(elem) {
+        this._elem = elem;
+        elem.onclick = this.onClick.bind(this);
+    }
+
+    addStart() {
+        let newItem = document.createElement('li');
+        newItem.textContent = listItems[Math.floor(Math.random() * listItems.length)];
+        ul.prepend(newItem);
+    }
+
+    addEnd() {
+        let newItem = document.createElement('li');
+        newItem.textContent = listItems[Math.floor(Math.random() * listItems.length)];
+        ul.append(newItem);
+    }
```

Comment:
You have 2 almost similar methods here (addStart, addEnd). Think about how you can make one method using the argument. (Don't Repeat Yourself)